HTML/1. intro.md

```diff
@@ -0,0 +1,37 @@
+# Intro
+
+![HTML](https://img.17qq.com/images/mhsgsnffsky.jpeg)
+
+HTML is a markup language, meaning that it is used to create webpages, so it's not like a programming language like Python
+
+___
+
+HTML is structured by tags, which make elements, here is an example of a tag - `<html>`
+
+And here is an example of an element `<html></html>`
+
+Each element has a opening tag and a closing tag, the opening tag has a `<` to begin and a `>` to end. The closing tag has a `<` to begin and a `/>` to end. Everything else between the opening and closing tags is either content or other nested elements
+
+There is an exception though, there are self closing tags, that don't have any content between them and they can be structered in two ways. With a `<` to begin and a `>` to end. And a `<` to begin and a `/>` to end.
```

Comment:
Maybe add examples of some void elements?

And remove the second structure, it's not HTML, it just works, since HTML doesn't throw exceptions or anything of the sort.